### PR TITLE
New package: lsp-plugins-1.2.5

### PR DESCRIPTION
**Pull request by subnut**
#### Testing the changes
- I tested the changes in this PR: **briefly**

#### Local build testing
- I built this PR locally for my native architecture, (x86_64-glibc)

### CODE REVIEW

**Comment by subnut** [comment]
Fixes #32991

**Comment by subnut** [comment]
According to the build logs, the build is failing on musl due to our outdated musl not having the `qsort_r` function, which was introduced in musl 1.2.3

@paper42 What should be done? Should I set `archs="~*-musl"`?

**Comment by paper42** [comment]
> According to the build logs, the build is failing on musl due to our outdated musl not having the `qsort_r` function, which was introduced in musl 1.2.3
> 
> @paper42 What should be done? Should I set `archs="~*-musl"`?

definitely no, either you can fix it with a patch or wait for #42425

**Comment by subnut** [comment]
Okay, I'll wait for #42425

File: srcpkgs/lsp-plugins/template [hunk 1]
```diff
@@ -0,0 +1,42 @@
+# Template file for 'lsp-plugins'
+pkgname=lsp-plugins
+version=1.2.5
```
Comment (zen0bit) [inline srcpkgs/lsp-plugins/template:3]: version=1.2.6

**Review by zen0bit — CHANGES_REQUESTED** [review]
New version 1.2.6 builded at tested on glibc 64bit...
(used changes of lv2...)

**Pull request merged**